tsl/test/sql/continuous_aggs.sql.in

```diff
@@ -1393,6 +1393,15 @@ GROUP BY time_bucket('1week', timec), location;
 
 SELECT * FROM mat_m1 ORDER BY 1, 2;
 
+-- Parallel planning for realtime continuous aggregate
+SET force_parallel_mode = 'on';
+SET parallel_setup_cost = 0;
+SET max_parallel_workers_per_gather = 1;
```

Comment:
Don't you have to use something larger than 1 to trigger the addition of a Gather or GatherMerge node and actually use a parallel plan?

Reply:
Reworked the tests!